src/main/java/frc/robot/subsystems/swerve/SwerveDrivetrain.java

```diff
@@ -204,8 +204,12 @@ public Rotation2d getGyroPitch() {
         return Rotation2d.fromDegrees(m_gyro.getPitch());
     }
 
-    public double getGyroRollPitch() {
-        return m_currentRoll - m_previousRoll;
+    public Rotation2d getGyroRoll() {
+        return Rotation2d.fromDegrees(m_gyro.getRoll());
+    }
+
+    public double getGyroPitchRate() {
+        return m_currentPitch - m_previousPitch;
```

Comment:
This isn't really the pitch rate unless it accounts for time.

Reply:
I can update the method to take time into account